src/settings.ts

```diff
@@ -89,7 +91,9 @@ const UPGRADE_FUNCTIONS: {[id: number]: (settings: ObloggerSettings) => void } =
         const newSettings = settings as ObloggerSettings_v2;
         if (newSettings) {
             newSettings.vaultVisible = true;
-            newSettings.clockVisible = true;
+            newSettings.clockVisible = false;
```

Comment:
Why have the clock off by default?

Reply:
it just seems appropriate that the default behavior is to have it off and that people should turn it on if they want it on. it seems like an "extra" feature.